src/features/events/hooks/useEventType.ts

```diff
@@ -15,8 +15,8 @@ export default function useEventType(
   );
 
   return loadItemIfNecessary(typeItem, dispatch, {
-    actionOnLoad: () => typeLoad(orgId),
```

Comment:
I'm honestly confused this ever even worked?? Like, would it not only have worked for cases where the org ID and event-type ID were the same??

Reply:
Exactly! So it was probably overlooked for that reason (testing it only with type 1, in org 1). And a big part of the problem was that there were no types to indicate that the arguments were incorrect.